src/context/SearchContext.jsx

```diff
@@ -0,0 +1,15 @@
+import { createContext, useContext, useState, useMemo } from 'react';
+
+const SearchContext = createContext();
+
+export const useSearch = () => useContext(SearchContext);
+
+export const SearchProvider = ({ children }) => {
+  const [searchQuery, setSearchQuery] = useState('');
+
+  const value = useMemo(() => ({ searchQuery, setSearchQuery }), [searchQuery]);
```

Comment:
👍

Reply:
Please, check your code after preparing the PR. Here you've added `useMemo` but in other hook you didn't

Reply:
Already checked.